### PR TITLE
chore: use stdout logging for docker container

### PR DESCRIPTION
**Pull request by darora**

### CODE REVIEW

File: ansible/files/postgresql_config/postgresql.conf.j2 [hunk 1]
```diff
@@ -428,39 +428,7 @@ effective_cache_size = 128MB
...
-					# 0 disables.
-#log_truncate_on_rotation = off		# If on, an existing log file with the
-					# same name as the new log file will be
-					# truncated rather than appended to.
-					# But such truncation only occurs on
-					# time-driven rotation, not on restarts
-					# or size-driven rotation.  Default is
-					# off, meaning append to existing files
-					# in all cases.
+include = '/etc/postgresql/logging.conf'
```
Comment (dragarcia) [inline ansible/files/postgresql_config/postgresql.conf.j2:431]: nit: Can we store it with the rest in `/etc/postgresql-custom` dir?

**Review by dragarcia — APPROVED** [review]

File: ansible/files/postgresql_config/postgresql.conf.j2 (hunk 1 above)
Comment (darora) [reply ansible/files/postgresql_config/postgresql.conf.j2:431]: The custom dir is currently used for config that can be dynamically updated (whether by user, or internal tooling). Going to leave as-is unless there's a strong objection.

**Comment by quangtd95** [comment]
@darora  may I ask if the latest docker image of supabase/postres applied this fix yet?

**Comment by darora** [comment]
Doesn't look like there's been a release since then; making one now https://github.com/supabase/postgres/pull/265

**Comment by quangtd95** [comment]
@darora  please inform us when the new image is ready to pull. really appreciate this

**Pull request merged**